Tiltfile.local

```diff
@@ -0,0 +1,27 @@
+# -*- mode: Python -*-
+
+"""
+* Frontend
+  * Language: Go
+  * Other notes: calls out to both `letters` and `numbers` microservices.
+* Letters
+  * Language: JavaScript
+  * Other notes: Uses yarn. Does a `yarn install` for package dependencies iff they have changed.
+* Numbers
+    * Language: Python
+    * Other notes: does a `pip install` for package dependencies. Reinstalls dependencies iff they have changed.
+"""
+
+local_resource(
```

Comment:
Does this technically have a resource dep on letters and numbers?

Reply:
Yeah, kinda. I feel like that's more important when the service makes requests at startup, and so fails and generates log noise if its dependencies aren't ready. For fe, it doesn't actually fail until request time (and then it crashes and doesn't restart, which is maybe more annoying in local_resource without a restart policy).